x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -594,3 +603,19 @@ export const reauthorizeTransformsHandler: FleetRequestHandler<
     return defaultFleetErrorHandler({ error, response });
   }
 };
+
+// Don't expose the whole SO in the API response, only selected fields
+const soToInstallationInfo = (pkg: PackageListItem | PackageInfo) => {
+  if ('savedObject' in pkg) {
+    const installationInfo = {
+      ...pick(pkg.savedObject, ['created_at', 'updated_at', 'namespaces', 'type']),
+      ...pkg.savedObject?.attributes,
```

Comment:
Could we map the attributes fields as well instead of the spread operator?

Reply:
I can map  the fields explicitly instead of using the spread operator, sure